packages/jaeger-ui/src/components/TracePage/TraceStatistics/index.css

```diff
@@ -29,3 +29,18 @@ limitations under the License.
 .test1893 {
   width: 99.999%;
 }
+
+.undefClass {
+  color: rgb(153, 153, 153);
+  font-style: italic;
+  cursor: default;
+}
+
+.MainTableData--tr {
+  border-top: 1px solid rgb(255, 255, 255);
+}
+
+.MainTableData--tr:hover {
+  background: rgb(250, 250, 250);
+  color: rgb(0, 0, 0);
+}
```

Comment:
Two styles above are namespaced by a suffix --TraceStatistics. Why wouldn't we use the same naming pattern?